example/lib/pages/epsg3006_crs.dart

```diff
@@ -0,0 +1,76 @@
+import 'dart:math';
+import 'package:flutter/material.dart';
+import 'package:flutter_map/flutter_map.dart';
+import 'package:flutter_map_example/widgets/drawer/menu_drawer.dart';
+import 'package:latlong2/latlong.dart';
+import 'package:proj4dart/proj4dart.dart' as proj4;
+
+typedef HitValue = ({String title, String subtitle});
+
+class EPSG3006Page extends StatefulWidget {
+  static const String route = '/crs_epsg3006';
+
+  const EPSG3006Page({super.key});
+
+  @override
+  State<EPSG3006Page> createState() => _EPSG3006PageState();
+}
+
+class _EPSG3006PageState extends State<EPSG3006Page> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(title: const Text('EPSG:3006 CRS')),
+      drawer: const MenuDrawer(EPSG3006Page.route),
+      body: Stack(
+        children: [
+          // OSM based, uses somewhat strange bounds.
+          // No access restrictions or fees in GetCapabilities as of 2024-10-15
+          FlutterMap(
+            options: MapOptions(
+                initialCameraFit: CameraFit.bounds(
+                  bounds: LatLngBounds(
+                    const LatLng(61.285991891313344, 17.006922572652666),
+                    const LatLng(61.279648385340494, 17.018309853620366),
+                  ),
+                ),
+                crs: Proj4Crs.fromFactory(
+                  code: 'EPSG:3006',
+                  proj4Projection: proj4.Projection.add(
+                    'EPSG:3006',
+                    '+proj=utm +zone=33 +ellps=GRS80 +towgs84=0,0,0,0,0,0,0 +units=m +no_defs +type=crs',
+                  ),
+                  origins: const [
+                    Point(-58122915.354077406, 19995929.885879364),
+                  ],
+                  resolutions: const <double>[
+                    4096,
+                    2048,
+                    1024,
+                    512,
+                    256,
+                    128,
+                    64,
+                    32,
+                    16,
+                    8,
+                    4,
+                    2,
+                    1,
+                    0.5,
+                    0.25,
+                    0.125
+                  ],
+                )),
+            children: [
+              TileLayer(
+                  tileBuilder: coordinateDebugTileBuilder,
+                  urlTemplate:
+                      'https://maps.trafikinfo.trafikverket.se/MapService/wmts.axd/BakgrundskartaNorden.gpkg?layer=Background&style=default&tilematrixset=Default.256.3006&Service=WMTS&Request=GetTile&Version=1.0.0&Format=image%2Fpng&TileMatrix={z}&TileCol={x}&TileRow={y}')
```

Comment:
What are the usage policy for this tile source? Source attribution would be a good think, too.

The tile source has long loading times for me. When I run the app on windows the tiles don't load but I got an exception. Not sure if this only happens on my machine though.
```console
======== Exception caught by image resource service ================================================
The following HandshakeException was thrown resolving an image codec:
Handshake error in client (OS Error: 
	CERTIFICATE_VERIFY_FAILED: unable to get local issuer certificate(../../../flutter/third_party/boringssl/src/ssl/handshake.cc:393))

When the exception was thrown, this was the stack: 
#0      _SecureFilterImpl._handshake (dart:io-patch/secure_socket_patch.dart:99:46)
#1      _SecureFilterImpl.handshake (dart:io-patch/secure_socket_patch.dart:143:25)
#2      _RawSecureSocket._secureHandshake (dart:io/secure_socket.dart:920:54)
#3      _RawSecureSocket._tryFilter (dart:io/secure_socket.dart:1049:19)
<asynchronous suspension>
#4      _RawSecureSocket._secureHandshake (dart:io/secure_socket.dart:928:9)
<asynchronous suspension>
#5      _RawSecureSocket._tryFilter (dart:io/secure_socket.dart:1049:13)
<asynchronous suspension>
URL: https://maps.trafikinfo.trafikverket.se/MapService/wmts.axd/BakgrundskartaNorden.gpkg?layer=Background&style=default&tilematrixset=Default.256.3006&Service=WMTS&Request=GetTile&Version=1.0.0&Format=image%2Fpng&TileMatrix=13&TileCol=466563&TileRow=111610
Fallback URL: null
Current provider: MapNetworkImageProvider()
====================================================================================================
flutter: HandshakeException: Handshake error in client (OS Error: 
flutter: 	CERTIFICATE_VERIFY_FAILED: unable to get local issuer certificate(../../../flutter/third_party/boringssl/src/ssl/handshake.cc:393))
flutter: HandshakeException: Handshake error in client (OS Error: 
flutter: 	CERTIFICATE_VERIFY_FAILED: unable to get local issuer certificate(../../../flutter/third_party/boringssl/src/ssl/handshake.cc:393))
flutter: HandshakeException: Handshake error in client (OS Error: 
flutter: 	CERTIFICATE_VERIFY_FAILED: unable to get local issuer certificate(../../../flutter/third_party/boringssl/src/ssl/handshake.cc:393))
flutter: HandshakeException: Handshake error in client (OS Error: 
flutter: 	CERTIFICATE_VERIFY_FAILED: unable to get local issuer certificate(../../../flutter/third_party/boringssl/src/ssl/handshake.cc:393))
flutter: HandshakeException: Handshake error in client (OS Error: 
flutter: 	CERTIFICATE_VERIFY_FAILED: unable to get local issuer certificate(../../../flutter/third_party/boringssl/src/ssl/handshake.cc:393))
flutter: HandshakeException: Handshake error in client (OS Error: 
flutter: 	CERTIFICATE_VERIFY_FAILED: unable to get local issuer certificate(../../../flutter/third_party/boringssl/src/ssl/handshake.cc:393))
```

Reply:
> What are the usage policy for this tile source? Source attribution would be a good think, too.

cc @arneke

Regarding CERTIFICATE errors, I've never experienced one, working on an Android emulator working on Windows.

Reply:
There's nothing published (I mention this at the top of the example in #1976). I included it to make it easy to reproduce the bug, was not really expecting this to end up on the Flutter Map website.

I can see if I can make a public service with dummy data available at some point, but not immediately and no guarantees about uptime or longevity.

Reply:
@arneke My initial point was to be able to test the fixes, for instance with the "3006" projection.
Any "non web-mercator" would have done the trick.
I've just removed "your 3006" from the PR.